### DISCUSSION TITLE
Welcome to achoz Discussions!

### DISCUSSION THREAD

**Discussion by kcubeterm** [kcubeterm/achoz#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cauerego** [comment]
i dunno... github looks awesome for the most part of the ux...

but i'd still have gone with fossil.

meaning i'm just worried on how to migrate out, when possible.

**Reply by kcubeterm** [reply]
Under the hood. I also love fossil and we have decided to release achoz with fossil. Migration is easy, no need of worry. Github is good for collaborations and many more.

**Reply by cauerego** [reply]
things like this discussion and issues aren't that straight forward, though.

at the same time, we might not need to migrate them...

let's give it time.
